_inc/client/security/index.jsx

```diff
@@ -91,7 +91,7 @@ export class Security extends Component {
 						isSearchTerm
 							? __( 'Security' )
 							: __(
-									'Keep your site safe with state-of-the-art security and receive notifications of technical problems.'
+									"Your site is protected by Jetpack. You'll be notified if anything needs attention."
```

Comment:
In other things you've switched to using an apostrophe instead of a single quote. Should we do that here too?

```suggestion
									'Your site is protected by Jetpack. You’ll be notified if anything needs attention.'
```